store.go

```diff
@@ -0,0 +1,128 @@
+package pluginapi
```

Comment:
Can we expose the `DriverName()` in this service, for when plugins inevitably have to make database-specific decisions?